frontend/src/modules/activity/components/activity-form-drawer.vue

```diff
@@ -0,0 +1,453 @@
+<template>
+  <app-drawer
+    v-model="isVisible"
+    :size="480"
+    :title="isEdit ? 'Edit activity' : 'Add activity'"
+  >
+    <template #content>
+      <div class="-mt-4">
+        <app-form-item
+          class="mb-4"
+          label="Member"
+          :validation="$v.member"
+          :required="true"
+          :error-messages="{
+            required: 'This field is required'
+          }"
+        >
+          <el-select
```

Comment:
Could we use `member-selection-dropdown` here? Since already has the logic to load members?